docs/GettingStartedDocs/UseNonDCAPDriverInAcc.md

```diff
@@ -0,0 +1,53 @@
+# Using the Non-DCAP Driver in an ACC VM
+
+ACC VMs come pre equipeed with the azure DCAP stack preinstalled. This can interfere with non-azure dcap clients, 
```

Comment:
Then I would say something like "ACC VMs come with a pre-installed Intel SGX DCAP driver. If you need to use the Intel Legacy Launch SGX driver instead you will need to disable this SGX DCAP driver first."